frontend/src/views/applications/ApplicationsHistoryView.vue

```diff
@@ -39,9 +95,13 @@ import { isEmpty } from 'lodash'
 import format from '@/utils/format'
 import CancelApplicationDialog from '@/components/applications/CancelApplicationDialog.vue'
 import ApplicationService from '@/services/applicationService'
+import AppFacilityFilter from '@/components/ui/AppFacilityFilter.vue'
+import { APPLICATION_ACTIVE_STATUS_CODES } from '@/utils/constants'
+
+const cardInfoMessage = 'If you are totally new in OFM you need to make a OFM application before apply for Supplementary Allowances.'
```

Comment:
I don't know if it's worth defining a constant outside of the component as well as a getter. You could just define this directly in created(). Also best to use CARD_INFO_MESSAGE.

Reply:
Fixed.